contrib/pyln-client/pyln/client/lightning.py

```diff
@@ -68,6 +68,10 @@ def __init__(self, v: Union[int, str, Decimal]):
 
         elif int(v) == v:
             self.millisatoshis = int(v)
+
+        elif isinstance(v, float) or isinstance(v, Decimal):
+            self.millisatoshis = int(v)
```

Comment:
Hm, makes me wonder if we should just go the full mile and maintain the internal value as `Decimal` and round on read, with `.round(), `.floor()` and `.ceil()` accessors so the caller can specify what is correct in their context.

Reply:
t.b.d. in a separate PR. I remove the init by float for now ...